IFC4x3/Properties/f/FaultSlipDirection_0_Rv9JVL5DTR1ugllbwXU4/Documentation.md

```diff
@@ -0,0 +1 @@
+Observed relative displacement direction of the hangingwall (top) to the footwall, e.g. up. Down, left, right or combinations (like top down - left).
```

Comment:
How are down, left and right defined?

Reply:
https://en.wikipedia.org/wiki/Fault_(geology)
![image](https://github.com/bSI-InfraRoom/IFC-Specification/assets/49076658/d126d488-5a87-4536-891f-b95eaa92ecc0)
relative displacement direction when looking from one block across the fault to the other block.
IFC label is fine

Reply:
Added figure to description.